fission-web-server/library/Fission/Web/Server/Internal/Production.hs

```diff
@@ -147,10 +147,16 @@ runInProd overrideVerbose action = do
   tlsManager <- HTTP.newManager tlsHttpSettings
 
   putStrLnIO "   💂 Configuring optional Sentry middleware"
-  condSentryLogger <- maybe (pure mempty) (Sentry.mkLogger RIO.LevelWarn) sentryDSN
+  condSentryLogger <- maybe (pure mempty) (Sentry.mkLogger host RIO.LevelWarn) sentryDSN
+
+  let
+    logOpts =
+      logOptions
+        |> setLogUseLoc  False
+        |> setLogUseTime True
 
   putStrLnIO "   📋 Setting up application logger"
-  withLogFunc (setLogUseTime True logOptions) \baseLogger -> do
+  withLogFunc logOpts \baseLogger -> do
```

Comment:
Let's make the logs easier to read:

# Before (prod)

<img width="696" alt="Screen Shot 2021-03-05 at 12 08 02 AM" src="https://user-images.githubusercontent.com/1052016/110086177-e817f900-7d46-11eb-9dac-9f72b091ce51.png">

# After (staging)

<img width="762" alt="Screen Shot 2021-03-05 at 12 07 05 AM" src="https://user-images.githubusercontent.com/1052016/110086188-eb12e980-7d46-11eb-8c4b-7974ffebbc2d.png">